README.md

```diff
@@ -9,3 +9,73 @@
 <a href="https://github.com/Frontify/arcade/blob/main/README.md" title="latest">
     <img alt="latest" src="https://img.shields.io/npm/v/@frontify/arcade/latest.svg" />
 </a>
+
+---
+
+# Updating Arcade
+
+## Tagging a new release
+
+Before we can use an updated version of Arcade in our main project, we need to tag a new release of the package.
```

Comment:
As this npm package is public and can be used in whatever project you want I don't think we have to mention "our main project" here. In my opinion this whole section is not really needed because `npm version` will create a new tag automatically.